sources/packages/web/src/views/institution/AppInstitution.vue

```diff
@@ -57,7 +61,9 @@
             <template v-for="(item, index) in menuItems" :key="index">
               <v-list-item :value="index" @click="item.command">
                 <v-list-item-title>
-                  <span class="label-bold">{{ item.label }}</span>
+                  <span class="label-bold" data-cy="settingsButton1">{{
```

Comment:
Why the `settingsButton1` was used here?
There will be many items rendered by the `v-for`, right?

Reply:
I was not completely sure if there will only be one CTA at this point. But you are right - i ll probably change it to just settings button. Will update it to be more generic and handle using index in the tests